### PR TITLE
F/player movement

### LINKED ISSUE #9: Player movement
* Player movement using WASD

### PR DESCRIPTION
**Pull request by tedgag**
Closes #9, also base project structure added. Movement is done using root motion and a blend tree to make the animation look more realistic. Note that player movement toward camera direction hasn't been implemented yet, as for now the camera is static

### CODE REVIEW

**Review by EvanDime — APPROVED** [review]
I was able to run and move the character. Animations looking really smooth. Felt wierd with the static camera but for now good stuff

File: .gitignore [hunk 1]
```diff
@@ -656,5 +655,5 @@ InitTestScene*.unity
 # Ionide (cross platform F# VS Code tools) working folder
 
 # End of https://www.toptal.com/developers/gitignore/api/unity,csharp,rider,vscode,visualstudio
-DRUNK/Library/AnnotationManager
```
Comment (ClaudiaLapalme) [inline .gitignore:659]: Why was this removed from the `.gitignore`?

File: Assets/Code/Scripts/PlayerMovementController.cs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            BoundCheck();
+        }
+
+        // Places the player on the opposite side of the plane if they try to leave it.
+        private void BoundCheck()
+        {
+            var newPos = new Vector3();
+            for (var i = 0; i < 3; i+=2)
+            {
+                if (transform.position[i] < _planePosition[i]-_planeDimensions[i]/2.0f)
```
Comment (ClaudiaLapalme) [inline Assets/Code/Scripts/PlayerMovementController.cs]: just to make sure: you want to do `_planePosition[i] - (-planeDimensions[i]/2)`?

File: Assets/Code/Scripts/PlayerMovementController.cs [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            ConstrainVelocity();
+            
+            _animator.SetFloat(VelocityXHash, _velocityX);
+            _animator.SetFloat(VelocityZHash, _velocityZ);
+            
+            BoundCheck();
+        }
+
+        // Places the player on the opposite side of the plane if they try to leave it.
+        private void BoundCheck()
```
Comment (ClaudiaLapalme) [inline Assets/Code/Scripts/PlayerMovementController.cs]: Is this function even necessary for our game?

**Review by ClaudiaLapalme — CHANGES_REQUESTED** [review]
Few questions

File: .gitignore (hunk 1 above)
Comment (tedgag) [reply .gitignore:659]: The library folder is already ignored as a whole, this was redundant

**Review by ClaudiaLapalme — APPROVED** [review]
